pkg/packagejson/parse.go

```diff
@@ -0,0 +1,37 @@
+package packagejson
+
+import (
+	"encoding/json"
+	"io"
+
+	"github.com/aquasecurity/go-dep-parser/pkg/types"
+	"golang.org/x/xerrors"
+)
+
+type packageDotJSON struct {
+	Name    string      `json:"name"`
+	Version string      `json:"version"`
+	SHA     string      `json:"_shasum"`
+	License interface{} `json:"license"`
+}
+
+func Parse(r io.Reader) ([]types.Library, error) {
```

Comment:
Is it possible that there are multiple libraries?
There is no need to implement a specific Interface, so a single return value is fine.